kano_qt_apps/lib/src/app.cpp

```diff
@@ -62,6 +62,7 @@ App::App(JSON_Object *app_object, std::shared_ptr<App> fb):
     this->icon = get_json_val<std::string>(app_object, "icon");
     this->hidden = get_json_val<bool>(app_object, "hidden");
     this->touch_only = get_json_val<bool>(app_object, "touch_only");
+    this->dashboard_less_mode = get_json_val<bool>(app_object, "dashboard_less_mode");
```

Comment:
Do we actually want the flag to be called `dashboard_less_mode` or should we be more generic and say something like `kill_other_apps` or `single_app_mode`? (well they are pretty lame names but you get my point)

Reply:
Calling it `single_app_mode` to signify that the app wants to run as the only app on the system because it's heavyweight.